opencti-platform/opencti-graphql/src/domain/backgroundTask-common.js

```diff
@@ -168,5 +169,5 @@ export const createListTask = async (context, user, input) => {
 };
 
 export const isTaskEnabledEntity = (entityType) => {
```

Comment:
We can add an information message for the user where the entity passed as a parameter is not managed on the back side.
What do you think ?

Reply:
The API handles that, we just didn't add it in front and it could hide the fact that we put the toolbar somewhere we don't want to.